content/docs/command-reference/exp/save.md

```diff
@@ -0,0 +1,63 @@
+# exp save
+
+Save the current project status as a [DVC experiment].
+
+[dvc experiment]: /doc/user-guide/experiment-management/experiments-overview
+
+## Synopsis
+
+```usage
+usage: dvc exp save [-h] [-q | -v] [-f]
+                   [--json] [-n <name>]
+                   [-I path]
+```
+
+## Description
+
+Provides a way to save the current status of your <abbr>project</abbr> as an
+<abbr>experiment</abbr> without polluting it with unnecessary commits, branches,
+directories, etc.
+
+Only files tracked by either Git or DVC are saved to the experiment. Use the
+`--include-untracked` (`-I`) option to explicitly include any untracked files in
+the experiment.
+
+[Review] your experiments with `dvc exp show`. Successful ones can be [made
+persistent] by restoring them via `dvc exp branch` or `dvc exp apply` and
+committing them to the Git repo. Unnecessary ones can be [cleared] with
+`dvc exp gc`.
+
+[review]: /doc/user-guide/experiment-management/comparing-experiments
+[made persistent]: /doc/user-guide/experiment-management/persisting-experiments
+[cleared]: /doc/user-guide/experiment-management/cleaning-experiments
```

Comment:
Instead of repeating the `dvc exp run` page, let's compare the two commands here.

Reply:
Just added a basic comparison. Let me know what do you think. It would be worth to emphasize the lightweight exp tracking with no pipeline setup requirements, although it might be worthwhile to have its own section, see my comment below.